tests/GUIUtils.lua

```diff
@@ -99,18 +99,21 @@ describe("Tests the GUI utilities as far as possible without mudlet", function()
         { "\27[38;2;120;134;94m", "<120,134,94:>"},
         { "\27[48;2;85;250;33m", "<:85,250,33>"},
         { "\27[38;2;120;134;94;48;2;85;250;33m", "<120,134,94:85,250,33>"},
```

Comment:
It is a way to describe characters in the ASCII character set table by column/row (see the diagram [here](https://en.wikipedia.org/wiki/ASCII) on Wikipedia) where each column is 16 characters so this means column 3 row 10 which is `:`... :grin: